sale_invoice_customs/model/stock_picking_out.py

```diff
@@ -0,0 +1,142 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    OpenERP, Open Source Management Solution
+#    This module copyright (C) 2014 Therp BV (<http://therp.nl>).
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+from openerp.osv import fields, orm, osv
+from openerp.tools.translate import _
+from openerp.tools.float_utils import float_round
+
+
+class stock_picking(orm.Model):
+    _inherit = 'stock.picking'
+    _columns = {
+        'customs_invoice_id': fields.many2one(
+            'account.invoice', 'Customs Invoice',
+            ),
+    }
+
+
+class stock_picking_out(orm.Model):
+    _inherit = 'stock.picking.out'
+    _columns = {
+        'customs_invoice_id': fields.many2one(
+            'account.invoice', 'Customs Invoice',
+            ),
+    }
+
+    def create_customs_invoice(self, cr, uid, ids, context=None):
+        picking = self.browse(cr, uid, ids, context=context)[0]
+        if picking.sale_id:
+            sale = picking.sale_id
+        else:
+            raise osv.except_osv(_('This delivery is not derived from a sale'),
+                                 _('To create a customs invoice, \
```

Comment:
those backslashes create weird string in your translation files (because the whole python indentation is used as part of the string) - better have one string per line